packages/bridge/src/runtime/composables/ssr.ts

```diff
@@ -14,5 +14,6 @@ export function useRequestHeaders (include?: any[]) {
 }
 
 export function useRequestEvent (nuxtApp: NuxtAppCompat = useNuxtApp()): H3Event {
-  return nuxtApp.ssrContext?.event as H3Event
+  if (nuxtApp.ssrContext?.event) { return nuxtApp.ssrContext.event }
+  return createEvent(nuxtApp.ssrContext?.req, nuxtApp.ssrContext?.res)
```

Comment:
What about caching the event?

```suggestion
  nuxtApp.ssrContext.event = createEvent(nuxtApp.ssrContext?.req, nuxtApp.ssrContext?.res)
  return nuxtApp.ssrContext.event
```

Reply:
As usual, I also had thoughts about it, but in this case direct call of ssrContext.event returns event/undefined depending on was 'useRequestEvent' called or not. Maybe we should cache, but not in .event, like ._event?